Makefile

```diff
@@ -120,6 +120,14 @@ builder-publish:
 olm-verify:
 	hack/dockerized "./hack/olm.sh verify"
 
+current-dir := $(shell pwd)
+
+build-prom-spec-dumper:
```

Comment:
(optional) suggestion: Maybe put this into a separate Makefile in subdir `/hack/prom-rule-ci`?

Reply:
I don't have a particular objection about moving it there. Why you think it will be better in a separate Makefile?

Reply:
(though I wouldn't want to block this PR from getting merged on it right now because it fixes a bug)

Reply:
I don't want to block the merge, I just didn't have time yet to have a look. And furthermore I don't yet understand what it does honestly.

Reply:
@dhiller it ensures that we don't have syntax or configuration errors in our prometheus rules.
`promtool` is shipped with the official Prometheus container, but it can only read files (no option to work from stdin directly), so first, we extract the prometheus rules for kubevirt and dump them to a file. Then, we pass it to `promtool` for linting.
Maybe in the future, we can also leverage this for unt-testing the rules as well...